service/resource/deployment/types.go

```diff
@@ -1,27 +1,14 @@
 package deployment
 
-import providerv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
-
-func newNodes(azureNodes []providerv1alpha1.AzureConfigSpecAzureNode) []node {
-	var ns []node
-	for _, n := range azureNodes {
-		ns = append(ns, newNode(n))
-	}
-	return ns
-}
-
 type node struct {
-	providerv1alpha1.AzureConfigSpecAzureNode
```

Comment:
No embedding. Only VMSize field will be left in AzureConfigSpecAzureNode, so it's just copied.